.github/ISSUE_TEMPLATE/proposal.yaml

```
@@ -0,0 +1,45 @@
+name: 💡 Proposal / Feature request
+description: 
+title: ""
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this proposal / feature reqeust
+  - type: checkboxes
+    id: preflight
+    attributes:
+      label: Preflight Checklist
+      options:
+      - label:
+          I could not find a solution in the existing issues, docs, nor discussions
+        required: true
+      - label:
+          I have joined the [ZITADEL chat](https://zitadel.com/chat)
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe your problem
+      description: Is your proposal / feature request related to a problem. Please describe it.
```

We don't care if it doesn't solve a real problem, right?
```suggestion
      description: Please describe your problem this proposal / feature is supposed to solve.
```